exercises/concept/luigis-luscious-lasagna/.docs/instructions.md

```diff
@@ -0,0 +1,30 @@
+# Instructions
+
+In this exercise you're going to write some code to help you cook a brilliant lasagna from your favorite cooking book.
+
+Your housemate is still busy at the office, and wants to know what time the lasagna will be ready, so that she can make sure she is back in time.
```

Comment:
Is this the same introduction text as the lasagna exercise? If so, you could consider slightly tweaking the text a bit. Maybe you've found a new favorite recipe?

Reply:
Ok, I've updated the text to continue the story a bit.